ara/cli/playbook.py

```diff
@@ -18,15 +18,25 @@
 from cliff.lister import Lister
 from cliff.show import ShowOne
 from cliff.command import Command
-from ara import models, utils
+from ara import models
 from ara.models import db
+from ara.fields import Field
+
+LIST_FIELDS = (
```

Comment:
Maybe, but only for `Duration` and `Complete`, because there are additional fields in `SHOW_FIELDS`.   I think that may be an unnecessary optimization.